src/gui/qgsmaptoolcapture.cpp

```diff
@@ -467,7 +467,7 @@ void QgsMapToolCapture::cadCanvasMoveEvent( QgsMapMouseEvent *e )
   {
     if ( !mCurrentShapeMapTool )
     {
-      emit messageEmitted( tr( "Cannot capture a shape without a shape tool defined" ), Qgis::MessageLevel::Warning );
+      emit messageEmitted( tr( "Cannot capture a shape without defining an option in the Shape Digitizing Toolbar" ), Qgis::MessageLevel::Warning );
```

Comment:
```suggestion
      emit messageEmitted( tr( "Cannot capture a shape without selecting an option in the Shape Digitizing Toolbar" ), Qgis::MessageLevel::Warning );
```

?

Reply:
Actually the whole string is awkward. Maybe "Select a option from the shape digitizing toolbar in order to capture shapes"